roles/pip/vars/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 pip_python_exec: "{{ pip_python3 | bool | ternary('python3', 'python') }}"
+pip_exec: "{{ pip_python3 | bool | ternary('pip3', 'pip') }}"
```

Comment:
ok, surprised that this isn't handled by the python that you installed it wiht.

Reply:
It's because of the altinstall. We keep the system level pip executable.